src/display/api.js

```diff
@@ -2085,6 +2096,10 @@ class WorkerTransport {
       isCompressed: params.cMapPacked,
     });
 
+    this.StandardFontDataFactory = new params.StandardFontDataFactory({
+      baseUrl: params.standardFontDataUrl,
+    });
```

Comment:
The new parameters, i.e. `standardFontDataUrl` and `StandardFontDataFactory`, should be added to https://github.com/mozilla/pdf.js/blob/00b4f86db301546d21d5dc79c61e2a9a829667a4/src/display/api.js#L109-L183